### PR TITLE
Fill general usage section in readme

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,56 @@ composer require yiisoft/form-model
...
+    #[Length(min: 8, lessThanMinMessage: 'Is too short.')]
+    private ?string $password = null;    
+    #[Safe]
+    private bool $rememberMe = false;
+}
+```
+
+Fill it with data and validate using [form hydrator](docs/guide/en/form-hydrator.md):
+
+```php
```
Comment (arogachev) [inline README.md:59]: Show usage in controller.

File: README.md [hunk 2]
```diff
@@ -31,12 +31,56 @@ composer require yiisoft/form-model
...
-2. Fill it with data and validate (FormHydrator)
-3. Display it (show both Field and FieldFactory!).
-4. ValidationRulesEnricher
+$isValid = $formHydrator->populateAndValidate($formModel, $request);
+$result = $form->getValidationResult();
+```
+
+Display it using [pure fields](docs/guide/en/pure-fields.md):
+
+```php
```
Comment (arogachev) [inline README.md:77]: Show usage in the view.

File: README.md [hunk 3]
```diff
@@ -31,12 +31,71 @@ composer require yiisoft/form-model
...
+use Yiisoft\FormModel\Field;
+use Yiisoft\FormModel\FormModel;
 
-1. Define a model (FormModelInterface, FormModel, Safe)
-2. Fill it with data and validate (FormHydrator)
-3. Display it (show both Field and FieldFactory!).
-4. ValidationRulesEnricher
+if (!empty($errors)) {
+    foreach ($errors as $property => $errorMessage) {
+        // Display an error message.
```
Comment (samdark) [inline README.md:88]: Like this?

```suggestion
        // Display an error message.
        <p><?= Html::encode($errorMessage) ?></p>
```

**Review by samdark — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -31,12 +31,71 @@ composer require yiisoft/form-model
...
+{
+    #[Required]
+    #[Length(min: 4, max: 40, lessThanMinMessage: 'Is too short.', greaterThanMaxMessage: 'Is too long.')]
+    #[Email]
+    private ?string $login = null;
+    #[Required]
+    #[Length(min: 8, lessThanMinMessage: 'Is too short.')]
+    private ?string $password = null;    
+    #[Safe]
+    private bool $rememberMe = false;
```
Comment (vjik) [inline README.md:53]:
```suggestion
    #[Required]
    #[Length(min: 4, max: 40, skipOnEmpty: true)]
    #[Email(skipOnEmpty: true)]
    private ?string $login = null;

    #[Required]
    #[Length(min: 8, skipOnEmpty: true)]
    private ?string $password = null;

    #[Safe]
    private bool $rememberMe = false;
```

File: README.md [hunk 5]
```diff
@@ -31,12 +31,71 @@ composer require yiisoft/form-model
...
+```
+
+Fill it with data and validate using [form hydrator](docs/guide/en/form-hydrator.md):
+
+```php
+use Psr\Http\Message\RequestInterface;
+use Yiisoft\FormModel\FormHydrator;
+use Yiisoft\FormModel\FormModel;
+
+final class PostController
```
Comment (vjik) [inline README.md:64]:
```suggestion
final class AuthController 
```

File: README.md [hunk 6]
```diff
@@ -31,12 +31,71 @@ composer require yiisoft/form-model
...
+```php
+use Psr\Http\Message\RequestInterface;
+use Yiisoft\FormModel\FormHydrator;
+use Yiisoft\FormModel\FormModel;
+
+final class PostController 
+{
+    public function login(RequestInterface $request, FormHydrator $formHydrator): ResponseInterface
+    {
+        /** @var FormModel $formModel */
```
Comment (vjik) [inline README.md:68]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -31,12 +31,71 @@ composer require yiisoft/form-model
...
+        if ($formHydrator->populateFromPostAndValidate($formModel, $request)) {
+            $errors = $formModel->getValidationResult()->getErrorMessagesIndexedByProperty();
+        }
+        
+        // You can pass $formModel and $errors to the view now.
+    }
+}
+```
+
+Display it using [pure fields](docs/guide/en/pure-fields.md) in the view:
```
Comment (vjik) [inline README.md:80]:
```suggestion
Display it using [fields](docs/guide/en/displaying-fields.md) in the view:
```

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**